packages/storybook/stories/Examples/TeamsInterop/snippets/Lobby.snippet.tsx

```diff
@@ -51,21 +52,32 @@ export const Lobby = (props: LobbyProps): JSX.Element => {
 
       <div
         style={{
-          zIndex: 0,
-          textAlign: 'center',
-          width: '50%',
-          height: '25%',
-          overflow: 'none',
-          margin: 'auto',
           position: 'absolute',
-          top: 0,
-          left: 0,
-          bottom: 0,
-          right: 0
+          top: '50%',
+          left: '50%',
+          transform: 'translate(-50%, -50%)',
+          marginTop: '-34px'
```

Comment:
Avoid px value here and use `rem` instead https://pixelsconverter.com/px-to-rem

Reply:
The other alternative to avoid having to offset by (presumably) half the height of the contents, is to use something like a stack to do it for you https://developer.microsoft.com/en-us/fluentui#/controls/web/stack

Reply:
Yes this is exactly what I was trying to do. Let me try out using a vertical stack here